src/utils/git.cpp

```diff
@@ -55,7 +55,7 @@ void Utils::Git::commitCurrentNoteFolder() {
     if (!executeGitCommand(git, QStringList{"init"}, process) ||
         !executeGitCommand(git, QStringList{"config", "commit.gpgsign", "false"}, process) ||
         !executeGitCommand(git, QStringList{"add", "-A"}, process) ||
-        !executeGitCommand(git, QStringList{"commit", "-m", "QOwnNotes commit"}, process)) {
+        !executeGitCommand(git, QStringList{"commit", "-m", "QOwnNotes commit"}, process, false, true)) {
```

Comment:
Showing an error dialog here will cause the user to be spammed with error messages every X seconds. And if the user isn't in front of a computer there maybe will be 1000s of dialogs 😅

Reply:
That one does not show up with error messages, the second boolean parameter means, that the return value is ignored.

Reply:
The errors are now shown in the status bar. So there will be no flood of dialogs.

Reply:
> ~Showing an error dialog here will cause the user to be spammed with error messages every X seconds. And if the user isn't in front of a computer there maybe will be 1000s of dialogs~
> 
> Ah, new parameter

Yes, to distinguish between "allowed/accepted" errors and the ones which are fatal.